app/views/mailing_list/steps/_already_qualified.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>We're sorry, but to sign up for our emails, you need to be interested in training to teach</h1>
```

Comment:
Is 'interested in' definitive enough @EllieNodder ?

Should we say something like

* 'We're sorry, but our emails are for people who aren't (already) qualified teachers'
* 'We're sorry, but our emails are for people who haven't qualified to teach yet'
* 'We're sorry, but our emails are for people who want to train to teach'

But I'm happy for you to ignore and leave as is!